doc/README-ENCDataVault.md

```diff
@@ -17,10 +20,12 @@ The decryption test is done on a 32-bit magic value, meaning that occasional
 false positives can occur. Cracked passwords for those false positives will not
 decrypt the vault correctly.
 
-The format has been tested on Sony ENCDataVault lite 6.2.13, SanDisk 
-SecureAccess 3.02 and ENC DataVault 7.1.1W.
+The format using the key derivation algorithm based on MD5, has been tested on 
+Sony ENCDataVault lite 6.2.13, SanDisk SecureAccess 3.02 and ENC DataVault 
+7.1.1W. The format using PBKDF-SHA256, has been tests on PrivateAccess 6.3.6W 
+and ENCDataVault 7.2.1.
```

Comment:
I think the commas before "has" should be removed, and in the second instance "tests" replaced with "tested".